go-packages/meep-sessions/session-mgr.go

```diff
@@ -114,3 +125,56 @@ func (sm *SessionMgr) Authorizer(inner http.Handler) http.Handler {
 		}
 	})
 }
+
+// StartSessionWatchdog -
+func (sm *SessionMgr) StartSessionWatchdog(handler SessionTimeoutHandler) {
+	// Update Watchdog callback function
+	sm.wdHandler = handler
```

Comment:
In relation to my next comment
- the handler should set inside the below `if !sm.wdStarted` statement - this way it's not possible to swap the handler at runtime. To swap user must stop and restart.
- if this function is called with a nil handler, we should simply return with an error before doing any further processing.